packages/video-intelligence/README.md

```diff
@@ -0,0 +1,25 @@
+# NodeJS Client for Google Cloud Video Intelligence API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
```

Comment:
This could qualify as a nit-pick, but the official spelling of Node is: "Node.js".